.gitignore

```diff
@@ -1,33 +1,70 @@
-# built application files
+# Built application files
 *.apk
 *.ap_
+*.aab
 
-# files for the dex VM
+# Files for the ART/Dalvik VM
 *.dex
 
 # Java class files
 *.class
 
-# generated files
+# Generated files
 bin/
 gen/
+out/
+
+# Gradle files
+.gradle/
+build/
 
 # Local configuration file (sdk path, etc)
 local.properties
 
-# Eclipse project files
-.classpath
-.project
-
 # Proguard folder generated by Eclipse
 proguard/
 
-# Intellij project files
+# Log Files
+*.log
+
+# Android Studio Navigation editor temp files
+.navigation/
+
+# Android Studio captures folder
+captures/
+
+# IntelliJ
 *.iml
-*.ipr
-*.iws
-.idea/
+.idea/workspace.xml
```

Comment:
Can you replace this and the lines below with 
```
.idea/
```
?

Reply:
I updated the .gitignore with the replace.
Please review commit 
1a3875e